tests/test_cases/test_cocotb/test_cocotb_35.py

```diff
@@ -0,0 +1,109 @@
+"""
+This file contains tests that use python 3.5-only syntax.
+
+This is likely to mean this file only tests `async def` functions.
+"""
+
+import cocotb
+from cocotb.triggers import Timer
+from cocotb.outcomes import Value, Error
+from cocotb.result import TestFailure
+
+
+class produce:
+    """ test helpers that produce a value / exception in different ways """
```

Comment:
Are spaces around docstrings common?
"Test" should be capitalized.

Reply:
Never too sure, I've seen conflicting opinions about this. Personally I find the spaces make it easier to read, but I'm happy to remove them

Reply:
I like the spaces, but feel free to do it however you'd like your code to look.

Reply:
Leaving as is, but will capitalize